en-US/E.xml

```diff
@@ -89,6 +89,19 @@
 			</listitem>
 
 		</varlistentry>
+		<varlistentry id="EOL">
+		 <term>EOL</term>
+			<listitem>
+			 <para>
+				 Initialism for "end of line" (n.) and "end-of-line" (adj.)
```

Comment:
I'd separate these. It'll make it easier when we start using more automation if word classes are on different lines, and it'll bring it inline with what I've been trying to do so far. e.g.,

"_adj., n._ Correct. Do not use 3D, 3-d, or other variations."

Reply:
Done